src/current/_includes/releases/v23.1/v23.1.12.md

```diff
@@ -0,0 +1,168 @@
+- Fixed an error on the [SQL Activity page](https://www.cockroachlabs.com/docs/v23.1/ui-overview#sql-activity) when there was a workload, and then the workload stopped so that no queries ran against the database in the last hour. [#111496][#111496]
+- The [**Jobs** table](https://www.cockroachlabs.com/docs/v23.1/ui-jobs-page) will now correctly display timestamps for creation, last modified, and the completed time fields. [#111901][#111901]
+
+<h3 id="v23-1-12-bug-fixes">Bug fixes</h3>
+
+- Fixed a nil dereference panic during node startup that could be caused by an incorrect initialization order. [#109684][#109684]
+- Fixed a bug where the [`BEGIN` statement](https://www.cockroachlabs.com/docs/v23.1/begin-transaction) incorrectly displayed a transaction ID in telemetry logs. It now correctly shows no transaction ID, since there is no open transaction when `BEGIN` is executed.[#109840][#109840]
+- Fixed a bug that could cause a transaction performing multiple parallel foreign key checks to return a `concurrent txn use detected` error. [#109849][#109849]
+- Fixed a bug where dependencies on sequences from tables would be reported with the wrong value for the `classid` column in the `pg_catalog.pg_depend` table. [#110207][#110207]
+- The `Reset Sql Stats` button is now visible to users with the admin role on the DB Console. [#110256][#110256]
+- Fixed edge cases in decimal and float evaluation for division operators. `'NaN'::DECIMAL / 0` will now return `NaN` instead of a division-by-zero error, and `0 / 'inf'::DECIMAL` will return `0` instead of `0E-2019`. [#110296][#110296]
+- Fixed a DB Console issue where the `DROP_UNUSED` index recommendations produced by the table details page produced an invalid `DROP INDEX` statement. [#110453][#110453]
+- Executing two [`ALTER RANGE default CONFIGURE ZONE`](https://www.cockroachlabs.com/docs/v23.1/alter-range#configure-zone) statements on the same line no longer triggers an error. [#110337][#110337]
+- Fixed a buggy TTL descriptor repair by removing it. Previously, upgrading from v22.2.x to 23.1.9 incorrectly removed [TTL storage parameters](https://www.cockroachlabs.com/docs/v23.1/row-level-ttl#ttl-storage-parameters) from tables (visible via [`SHOW CREATE TABLE  <ttl-table>`](https://www.cockroachlabs.com/docs/v23.1/show-create)) while attempting to repair table descriptors. This resulted in the node that attempts to run the TTL [job](https://www.cockroachlabs.com/docs/v23.1/show-jobs.html) crashing due to a panic caused by the missing TTL storage parameters. Clusters currently on v22.2.x should **not** be upgraded to v23.1.9 and should be upgraded directly to v23.1.10 or later. For more information, see [Technical Advisory 110363](../advisories/a110363.html). [#110500][#110500]
+- Fixed a bug causing performance regression when disabling `sql.metrics.statement_details.enabled` which caused execution stats to be collected for all queries instead of the default one percent. [#109881][#109881]
```

Comment:
```suggestion
- Fixed a performance regression when disabling `sql.metrics.statement_details.enabled`, which caused execution stats to be collected for all queries instead of the default 1% of queries. [#109881][#109881]
```